docs/kpod-stats.1.md

```diff
@@ -17,6 +17,10 @@ Display a live stream of one or more containers' resource usage statistics
 
 Show all containers.  Only running containers are shown by default
 
+**--no-reset**
+
+Do not clear the terminal/screen in between reporting intervals
```

Comment:
It would be nice to add some examples.  Not a hold up for this PR though.

Reply:
Done